tests/conftest.py

```diff
@@ -22,6 +22,13 @@
     (50, 30)
 ]
 
+wrong_parameters = [
```

Comment:
Cool! We need to keep the number of parameterized tests to a minimum for test performance. What's a sensible place to test with these invalid values?